stdlib/sql/vertica.go

```diff
@@ -0,0 +1,170 @@
+package sql
+
+import (
+	"database/sql"
+	"fmt"
+	"time"
+
+	"github.com/influxdata/flux"
+	"github.com/influxdata/flux/codes"
+	"github.com/influxdata/flux/execute"
+	"github.com/influxdata/flux/internal/errors"
+	"github.com/influxdata/flux/values"
+)
+
+type VerticaRowReader struct {
+	Cursor      *sql.Rows
+	columns     []interface{}
+	columnTypes []flux.ColType
+	columnNames []string
+}
+
+func (m *VerticaRowReader) Next() bool {
+	next := m.Cursor.Next()
+	if next {
+		columnNames, err := m.Cursor.Columns()
+		if err != nil {
+			return false
+		}
+		m.columns = make([]interface{}, len(columnNames))
+		columnPointers := make([]interface{}, len(columnNames))
+		for i := 0; i < len(columnNames); i++ {
+			columnPointers[i] = &m.columns[i]
+		}
+		if err := m.Cursor.Scan(columnPointers...); err != nil {
+			return false
+		}
+	}
+	return next
+}
+
+func (m *VerticaRowReader) GetNextRow() ([]values.Value, error) {
+	row := make([]values.Value, len(m.columns))
+	for i, col := range m.columns {
+		switch col := col.(type) {
+		case bool, int, uint:
+			row[i] = values.New(int64(col.(int)))
+		case int64, uint64, float64:
+			row[i] = values.New(col)
+		case string:
+			row[i] = values.New(col)
+		case time.Time:
+			row[i] = values.NewTime(values.ConvertTime(col))
+		case []uint8:
+			fmt.Print("hack:", m.columnTypes[i])
```

Comment:
This got left behind? There shouldn't be any IO here.

Also, it seems that Vertica is based off of postgres, based on the usage of `PostgreColumnTranslateFunc`. But here it appears to be based off of mysql? Sorry not too familiar with Vertica.

Reply:
Vertica is based on Postgres, but I based my patch on the SAP HANA patch.  This fmt.Print appears to be an artifact from that.  I'm not sure this code would be reachable since Vertica uses 64-bit integer types.